src/esp/bindings/AttributesManagersBindings.cpp

```diff
@@ -185,11 +185,11 @@ void declareBaseAttributesManager(py::module& m,
             "search_str.")
                .c_str(),
            "search_str"_a = "", "contains"_a = true)
-      .def("remove_template_by_ID", &MgrClass::removeObjectByID,
+      .def("remove_template_by_id", &MgrClass::removeObjectByID,
            ("This removes, and returns the " + attrType +
             " template referenced by the passed ID from the library.")
                .c_str(),
-           "ID"_a)
+           "id"_a)
```

Comment:
This still cause flake8 to complain when used as a kwarg, at leasst the syntax is short and consistent.  May want to modify the arg name slightly though.

Reply:
That's odd, that shouldn't be there.